src/worldmap/camera.cpp

```diff
@@ -31,7 +31,8 @@ const float CAMERA_PAN_TIME_MAX = 0.52213f;
 
 namespace worldmap {
 
-Camera::Camera() :
+Camera::Camera(WorldMapSector& world_map_sector) :
+  m_world_map_sector(world_map_sector),
```

Comment:
`worldmap` shouldn't be separated, since it's used as one word.

Reply:
I changed it. I was unsure because of the capitalization in the class name.